src/UiPath.Workflow.Runtime/Runtime/CallbackWrapper.cs

```diff
@@ -6,33 +6,32 @@
 namespace System.Activities.Runtime;
 
 [DataContract]
-internal class CallbackWrapper
+public class CallbackWrapper
 {
     private static readonly BindingFlags bindingFlags = BindingFlags.Instance | BindingFlags.NonPublic | BindingFlags.Public | BindingFlags.DeclaredOnly | BindingFlags.Static;
 
     private string _callbackName;
     private string _declaringAssemblyName;
     private string _declaringTypeName;
-    private Delegate _callback;
     private ActivityInstance _activityInstance;
 
     protected internal CallbackWrapper() { }
 
     public CallbackWrapper(Delegate callback, ActivityInstance owningInstance)
     {
         ActivityInstance = owningInstance;
-        _callback = callback;
+        Callback = callback;
```

Comment:
The rename of the backing field is also a risk.

Reply:
Added back backing field _callback.